creator-node/src/index.js

```diff
@@ -25,6 +24,11 @@ const initAudiusLibs = async () => {
     config.get('ethNetworkId'),
     /* requiresAccount */ false
   )
+  const dataWeb3 = await AudiusLibs.Utils.configureWeb3(
+    config.get('dataProviderUrl'),
+    null,
```

Comment:
should this param be `config.get('dataNetworkId')`?

Reply:
@vicky-g  this is done on purpose actually. the network id is inferred here. also by leaving this out, we can manually evade the if condition that actually gets us into trouble with the utils configureWeb3 function check. kind of a hack but works for now 🤷